### DISCUSSION TITLE
Can't log in to Hubs Admin

### DISCUSSION THREAD

**Discussion by BonzoMartini** [mozilla/hubs#3426] [Hubs Cloud Questions] [answered]
Have checked the AWS admin email & verified with a test email.
I set up a different email and verified with AWS test email
Previously I set up Sendgrid in added those details in the Admin panel and verified emails etc.

SendGrid will only send emails to emails which are not @myDomain.com

MyHubs cloud can't be logged into because it won't send me emails.

Now what?

Please help

This was all because I wondered if using a new browser/clearing browser cache etc. would solver some of the other problems I'm having with Hubs Cloud. I have reset the Stack on AWS etc too. Every attempt to move forward moves me 1 space back

Browser Firefox, PC, Windows 10

**Comment by brianpeiris** [comment]
To confirm, is your stack currently setup to use SendGrid? 
In the SendGrid console, can you confirm that SendGrid is receiving the emails from your stack? 
And finally, just to be sure, have you checked your spam folder for the admin sign-in email?

**Reply by BonzoMartini** [reply]
Thankyou. 
Yes checked spam, nothing
I believe the Hub is set to use Sendgrid.
In the emails to a gmail account, from my Hubs, I can see that email is via sendgrid. 
( I cannot figure out where to see if Sendgrid is receiving emails)

**Reply by BonzoMartini** [reply]
is there some way to change the admin email address in the stack? i tried with an update, but it would not update and rolled back

**Reply by brianpeiris** [reply]
If sendgrid successfully sending hubs emails to your gmail account, it should also be able to send emails to the custom domain that your admin email is using. You may want to try confirming this on the sendgrid side.

If you would like to change the admin email address instead, that should be possible. What issue is it running into when you try to update the stack? You should see specific error messages in the "Events" tab (make sure you scroll down and click on "Load More" if necessary).

**Reply by BonzoMartini** [reply]
Yes i would have thought that sending to admin email  - but it won't send to the admin email address, so I must change admin email. Updating the stack with 'safe' email address ends up with UPDATE_ROLLBACK_COMPLETE

**Reply by BonzoMartini** [reply]
The 'safe' email is verified too...

**Reply by brianpeiris** [reply]
"UPDATE_ROLLBACK_COMPLETE" is the final status of the update. You'll need to scroll through the "Events" tab and find the reason for the rollback. You should see events that have a status of FAILED. You'll need to get that info before we can troubleshoot in more depth.

**Comment by BonzoMartini** [comment]
well it says
SESDomain UpdateInProgress (in Blue) 
SESDomain UpdateComplete
AppDbBudget UpdateComplete
expanded01 UPDATE_ROLLBACK_COMPLETE_CLEANUP_IN_PROGRESS
AppDbBudget DELETE_COMPLETE

**Reply by BonzoMartini** [reply]
so nothing has failed? the only thing I changed in the update was the email.

**Reply by BonzoMartini** [reply]
would it be the way to change the email address be with a "change set"

**Reply by brianpeiris** [reply]
I'll have to confirm that a stack update will actually change the admin email on the backend. I'm also surprised that you're seeing it say "UPDATE_ROLLBACK_COMPLETE" without any failures, so I'll try to confirm that's expected as well.

FYI, other alternatives are to figure out why SendGrid seemingly won't send emails to your existing admin email, and of course you can always delete and create the entire stack from scratch, unless you have reason to retain the existing stack.

**Comment by BonzoMartini** [comment]
Thankyou
SendGrid's IP seems to be showing up on blacklists, according to my email support. Gmail seems to have different set of 3rd party blacklists.
I would be happy to delete and remake the stack using the hubs guide no problemo, BUT will links to my existing rooms remain the same?

**Reply by brianpeiris** [reply]
Sorry, I should have been more clear about deleting your stack. It will definitely result in all your existing rooms, scenes and data being deleted. I was just offering it as a suggestion if you didn't particularly care about the existing stack.

**Reply by BonzoMartini** [reply]
so could i delete the stack and restore from backup?

**Comment by BonzoMartini** [comment]
ok good to know! So would rather Update the stack with a new email address - um, would a "change set" be a thing to do?

**Comment by BonzoMartini** [comment]
Ok I managed to update the Stack with my Gmail address as the admin email.
I cleared the cache etc. rebooted the PC. Went to my hub domain expandedfields.net to sign in, signed in - went to gmail, got the verify email clicked the link. Expandedfields.net changed to _"Verification Complete - Please close this browser window and return to hubs."_ - did that. 
Went to https://expandedfields.net/admin & get a message _"Sorry! Your account is not an admin. Return to home"_ So I am signed in as a user not the admin. Double checked the Stack parameters - the gmail address is the admin address. 

Any ideas

**Reply by emclaren** [reply]
Just double checking - have you gone through these steps to check your admin email address? https://hubs.mozilla.com/docs/hubs-cloud-aws-troubleshooting.html#then-what-is-my-hub-stacks-admin-email-address

**Comment by BonzoMartini** [comment, marked as answer]
<img width="593" alt="sorry yourt account is not an admin" src="https://user-images.githubusercontent.com/66266175/100349049-bec92d80-2fdf-11eb-93b2-dd32b0027536.png">

**Reply by robinkwilson** [reply]
Sounds like you fixed your issue with sending emails. Can you log into both admin email accounts from your HC instance? And see if they are admins.

**Reply by robinkwilson** [reply]
Still having this issue? Marking this as answer, "Unmark as answer" when you add more info!
